packages/ui/src/components/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -41,15 +43,11 @@ class DropdownProps {
   position = prop<string>({ type: String, default: '' })
   hoverOverTimeout = prop<number>({ type: Number, default: 30 })
   hoverOutTimeout = prop<number>({ type: Number, default: 200 })
-  boundaryBody = prop<boolean>({ type: Boolean })
   modelValue = prop<boolean>({ type: Boolean, default: false })
   disabled = prop<boolean>({ type: Boolean })
-  // Makes no sense
-  // fixed = prop<boolean>({ type: Boolean })
+  teleportTarget = prop<string>({ type: String, default: '#app' })
```

Comment:
Need a better name for this prop. We don't need to use 'teleport' work in this prop name.